skimage/feature/tests/test_blob.py

```diff
@@ -169,6 +170,26 @@ def test_blob_dog_excl_border():
     assert blobs.shape[0] == 0, msg
 
 
+@pytest.mark.parametrize('anisotropic', [False, True])
+@pytest.mark.parametrize('ndim', [1, 2, 3, 4])
+@pytest.mark.parametrize('function_name', ['blob_dog', 'blob_log'])
+def test_nd_blob_no_peaks_shape(function_name, ndim, anisotropic):
+    # uniform image so no blobs will be found
+    z = np.zeros((16,) * ndim, dtype=np.float32)
+    if anisotropic:
+        max_sigma = 8 + np.arange(ndim)
+    else:
+        max_sigma = 8
+    blob_func = getattr(feature, function_name)
+    blobs = blob_func(z, max_sigma=max_sigma)
+    if anisotropic:
+        # z.ndim coordinates and z.ndim sigmas
+        assert blobs.shape == (0, 2 * z.ndim)
+    else:
+        # z.ndim coordinates and 1 sigma
+        assert blobs.shape == (0, z.ndim + 1)
```

Comment:
```suggestion
    # z.ndim +  (z.ndim sigmas if anisotropic, only one sigma otherwise)
    expected_shape = 2 * z.ndim if anisotropic else z.ndim + 1
    assert blobs.shape == (0, expected_shape)
```

Reply:
thanks, this is clear, but concise